server/app/src/androidTest/java/sh/calaba/instrumentationbackend/actions/device/UiautomatorSwipeElement.java

```diff
@@ -50,6 +52,6 @@ public Result execute(String... args) {
 
     @Override
     public String key() {
-        return "uiautomator_seek_bar";
+        return "uiautomator_swipe_element";
```

Comment:
this is a breaking change, right?

Reply:
I guess yes, this will break my [PR](https://github.com/XINGMobile/android-app/pull/6070/files)

Reply:
Yes, if we want to be strict we should keep it and create a copy instead of renaming it. tbh I don't think there is anyone else using the step apart from us but it was added last year here https://github.com/calabash/calabash-android-server/pull/119/files so it is probably safer to deprecate the action instead. I'll do the changes.